testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -310,39 +345,43 @@ impl Swarm for LocalSwarm {
     }
 
     fn full_nodes<'a>(&'a self) -> Box<dyn Iterator<Item = &'a dyn FullNode> + 'a> {
-        Box::new(self.full_nodes.values().map(|v| v as &'a dyn FullNode))
+        Box::new(self.fullnodes.values().map(|v| v as &'a dyn FullNode))
     }
 
     fn full_nodes_mut<'a>(&'a mut self) -> Box<dyn Iterator<Item = &'a mut dyn FullNode> + 'a> {
         Box::new(
-            self.full_nodes
+            self.fullnodes
                 .values_mut()
                 .map(|v| v as &'a mut dyn FullNode),
         )
     }
 
     fn full_node(&self, id: PeerId) -> Option<&dyn FullNode> {
-        self.full_nodes.get(&id).map(|v| v as &dyn FullNode)
+        self.fullnodes.get(&id).map(|v| v as &dyn FullNode)
     }
 
     fn full_node_mut(&mut self, id: PeerId) -> Option<&mut dyn FullNode> {
-        self.full_nodes.get_mut(&id).map(|v| v as &mut dyn FullNode)
+        self.fullnodes.get_mut(&id).map(|v| v as &mut dyn FullNode)
     }
 
-    fn add_validator(&mut self, _id: PeerId) -> Result<PeerId> {
+    fn add_validator(&mut self, _version: &Version, _template: NodeConfig) -> Result<PeerId> {
         todo!()
     }
 
     fn remove_validator(&mut self, _id: PeerId) -> Result<()> {
         todo!()
     }
 
-    fn add_full_node(&mut self, _id: PeerId) -> Result<()> {
-        todo!()
+    fn add_full_node(&mut self, version: &Version, template: NodeConfig) -> Result<PeerId> {
```

Comment:
You know we should really decide someday on the standard of `fullnode` vs `full_node`

Reply:
yeah seriously, even i do it inconsistently